blockchain/chain.go

```diff
@@ -1687,21 +1693,22 @@ func New(config *Config) (*BlockChain, error) {
 		return nil, AssertError("blockchain.New timesource is nil")
 	}
 
-	// Generate a checkpoint by height map from the provided checkpoints.
-	params := config.ChainParams
+	// Create checkpoint variables. Leave empty when disabled.
```

Comment:
Any reason this comment was changed?  It seems like it more clearly stated the intention before.  "Create checkpoint variables" doesn't actually tell me what the intent is in my opinion.